docs/configuration/overview/overview.rst

```diff
@@ -45,9 +45,11 @@ flags_path
   *(optional, string)* The file system path to search for :ref:`startup flag files
   <operations_file_system_flags>`.
 
-statsd_local_udp_port
```

Comment:
shall we keep this until it's removed from the code? (and add deprecation warning)